lib/Parse.ml

```diff
@@ -1,68 +0,0 @@
-(** Some complicated stuff to get the custom syntax errors out of Menhir's Incremental
-    API *)
-
-open Errors
-
-let parse parse_fun lexbuf =
-  (* see the Menhir manual for the description of
-     error messages support *)
-  let open MenhirLib.General in
-  let module Interp = Parser.MenhirInterpreter in
-  let _ = Stack.push lexbuf Lexer.include_stack in
-  let input _ =
```

Comment:
I think you can get rid of the _ and the wrapping () and the empty () and use partial application